File: 프로그래머스/0/181906. 접두사인지 확인하기/접두사인지 확인하기.js
function solution(my_string, is_prefix) {
    for(let i = 0; i < my_string.length; i++){
        let str = '';
        str = my_string.substring(0,i+1);
        
        if(str == is_prefix) return 1;
    }
    return 0;
}

File: 프로그래머스/0/181906. 접두사인지 확인하기/접두사인지 확인하기.cpp
#include <string>
#include <vector>

using namespace std;

int solution(string my_string, string is_prefix) {
    int answer = 0;
    
    for(int i = 0; i < my_string.length(); i ++){
        string str = my_string.substr(0,i+1);
        if(str == is_prefix){
            return 1;
        }
    }
    
    return 0;
}